libs/primitives/src/lib.rs

```diff
@@ -90,15 +90,20 @@ pub mod types {
 	pub type Index = u32;
 
 	/// A hash of some data used by the chain.
-	pub type Hash = sp_core::H256;
+	pub type Hash = <BlakeTwo256 as HashT>::Output;
```

Comment:
Do you mind just referencing `sp_runtime::traits::Hash` instead of importing it with an alias?

Reply:
Following the Nuno's comment, maybe importing `sp_runtime::traits` and then using `traits::Hash`, is still readable without the annoyance of writing the entire path in the type.

Reply:
matter of taste anyway!

Reply:
yeah I like that, @lemunozm. I am not fond of import alias, specially when there is no name conflict to resolve.